### ISSUE TITLE
How to set the contamination range for function autoEstCont?

### ISSUE THREAD

**Issue by honghh2018** [constantAmateur/SoupX#65]
Hi developer,
     Thanks for the nice tools to process ambient mRNA.
      Even though the process and vignitte is clear for me, the contaminationRange  of  autoEstCont's parameter had me confused.
      how to set the contaminationRange  for my own data if i consider the ambient rate of my 10X scRNA data as 2%-5% or so.
      What mean the contaminationRange default value between 0.01 and 0.8 ? 
      Any kindly help would be appreciated.
      Best,

**Comment by constantAmateur** [comment]
This parameter just sets the range of plausible values for the contamination fraction.  A solution will only be considered valid if it falls within this range.

For most datasets, you could set this to c(0,1) (equivalent to no constraint) and it wouldn't make a difference.  Occasionally, datasets can have a big spike of estimates at exactly 0, which the defaults prevent being chosen as the preferred solution.

Basically, it's fine to ignore it and use the defaults.  Unless your data produces some weird multi-modal distribution of estimates of the contamination when you run `autoEstCont`, in which case you can use this parameter to filter out parts of the parameter space for the contamination fraction.
